mkosi/__init__.py

```diff
@@ -2493,8 +2494,35 @@ def invoke_apt(
         INITRD="No",
     )
 
-    run_workspace_command(args, root, cmdline, network=True, env=env)
+    # Ensure apt inside the image can see the local mirror outside of it
+    if args.local_mirror:
+        mirror : Optional[str] = args.local_mirror
+    else:
+        mirror = args.mirror
+    if mirror and mirror.startswith("file:"):
```

Comment:
With the assumptions from the above comment this can be just 
```suggestion
    if mirror.startswith("file:"):
```

Would it also make sense to ensure in `load_args` that on Debian of the argument of `--local-mirror` start with `file:`?

Reply:
> Would it also make sense to ensure in load_args that on Debian of the argument of --local-mirror start with file:?

I don't think so, technically a local mirror might be on a different machine on the same local network, with the same principle applying that the local one is used only for the build, but not configured in the image